helpers/classes.py

```diff
@@ -0,0 +1,153 @@
+import re
+from .config import LMS_URL
+
+class Block:
+    '''
+    Represents each block in a course
+    '''
+    def __init__(self, block_dict) -> None:
+        self.usage_key = block_dict.get('id', '')
+        self.id = block_dict.get('block_id', '')
+        self.lms_url = block_dict.get('lms_web_url', '')
+        self.studio_url = None
+        self.type = block_dict.get('type', '')
+        self.display_name = block_dict.get('display_name', '')
+        self.children = block_dict.get('children', None)
+        self.parent = None
+        self.data = None
+
+    def __str__(self) -> str:
```

Comment:
```suggestion
    def __repr__(self) -> str:
```

This will also work for debugging.